### ISSUE TITLE
Compile fails to add 'required' to net7.0 generated code

### ISSUE THREAD

**Issue by Dzoay** [jamescourtney/FlatSharp#348]
Just cloned the latest code and rebuilt the samples with these project settings:
```xml
<TargetFramework>net7.0</TargetFramework>
<LangVersion>latest</LangVersion>
```
Building this generates code as follows that is an error due to the lack of ```required``` 
```csharp
#if NET7_0_OR_GREATER
            public override System.Collections.Generic.IList<System.String> Message
#else
            public override System.Collections.Generic.IList<System.String> Message
#endif
```

Am I missing some configuration here?

Further investigations:
- The required test (Required_Attribute_OnSerialize) passes when running specifically net7.0 and debugging indicates correct adding of the required keyword
- I am not sure this is relevant but FlatSharp.Compiler 7.0.0 nuget package does not contain net7.0 tools only net6.0

**Comment by ben3441** [comment]
I have the same issue after upgrading to .net 7

**Comment by jamescourtney** [comment]
This looks like a bug. I'm sorry for the miss here. I will take a look this evening. Thanks for pointing this out!

**Comment by jamescourtney** [comment]
My best guess at the moment is that this is because your build machine does not have .NET 7 installed (as all of the test environments do...). This means that when Flatsharp runs, it sees "non-required" properties. You can mitigate this quickly by installing NET7 on your build machines or local machines.

**Comment by jamescourtney** [comment]
Fixed in 7.0.1
